### PR TITLE
front: fix incorrect time in conflict card

### LINKED ISSUE #8548: Conflict times are not correct
### What happened?

I have a conflict here between 5h and 6h:
image

But my freight train leaves between 6 and 7:
image

### What did you expect to happen?

Trains are always on time, so i'm guessing the conflict should follow that time

### How can we reproduce it (as minimally and precisely as possible)?

1. create a scenario with conflicting trains

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Google Chrome

### OSRD version (top right corner `Account` button > `Informations`)

c885d29

### PR DESCRIPTION
**Pull request by RomainValls**
close #8548 

In formatToIsoDate function, we now convert the date with dayjs first, then apply the userTimeZone. 
It avoids double application of timezone offsets.

### CODE REVIEW

File: front/src/utils/date.ts [hunk 1]
```diff
@@ -82,8 +82,14 @@ export const dateTimeToIso = (inputDateTime: string) => {
...
   const userTimeZone = dayjs.tz.guess(); // Format : 'Europe/Paris'
+
+  let parsedDate = dayjs(date);
+
+  parsedDate = parsedDate.tz(userTimeZone);
+
   const format = formatDate ? 'D/MM/YYYY HH:mm:ss' : '';
-  return dayjs.tz(date, userTimeZone).format(format);
+
+  return parsedDate.format(format);
```
Comment (SharglutDev) [inline front/src/utils/date.ts]: I think you can do it shorter like that (and removed parsedDate logic) : 

```suggestion
  return dayjs(date).tz(userTimeZone).format(format);
```

**Review by SharglutDev — COMMENTED** [review]
Thank you for the fix. It works great but left one suggestion so the code can be shorter.

File: front/src/utils/date.ts (hunk 1 above)
Comment (RomainValls) [reply front/src/utils/date.ts]: Indeed, done 👍 !

**Review by SharglutDev — APPROVED** [review]
Lgtm :)

**Pull request closed without merging**